### ISSUE TITLE
Read more link Colour contrast fails

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#118]
![image](https://cloud.githubusercontent.com/assets/15111528/10998876/83c1841a-84ec-11e5-9144-17746c65988b.png)

**Comment by mr-bobz** [comment]
I think you are testing against the pattern color. I have made it lighter, should be fine now. Pls check again, tx

**Comment by christosptr** [comment]
verified
